### PR TITLE
[MPOM-298] Cleanup dependencyLocationsEnabled from MPIR configuration

### PR DESCRIPTION
**Pull request by slawekjaranowski**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -1209,9 +1209,6 @@ under the License.
       <plugin>
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-project-info-reports-plugin</artifactId>
-        <configuration>
-          <dependencyLocationsEnabled>false</dependencyLocationsEnabled><!-- waiting for MPIR-267 -->
-        </configuration>
```
Comment (michael-o) [inline pom.xml:1214]: The issue has been autoclosed, Is this the right thing to do?

Comment (slawekjaranowski) [reply pom.xml:1214]: Yes was removed in https://issues.apache.org/jira/browse/MPIR-367
Wrong comments

**Review by michael-o — APPROVED** [review]

**Review by slachiewicz — APPROVED** [review]

**Comment by hboutemy** [comment]
can you explain what it is about, please?

File: pom.xml (hunk 1 above)
Comment (hboutemy) [reply pom.xml:1214]: not wrong comment: the real issue is MPIR-267
but I discover that the problem was "solved" by removing the feature in MPIR-367

then yes, no need to keep the configuration, thanks for noticing this

**Review by hboutemy — APPROVED** [review]

**Comment by jira-importer** [comment]
Resolve #255

**Pull request merged**